.github/workflows/linux.yml

```diff
@@ -141,7 +141,8 @@ jobs:
           cd build
           cmake -DCMAKE_INSTALL_PREFIX=$HDF5_DIR -DHDF5_ENABLE_PARALLEL=ON -DHDF5_ENABLE_THREADSAFE=ON \
             -DHDF5_ALLOW_UNSUPPORTED=ON -DBUILD_TESTING=OFF -DHDF5_BUILD_HL_LIB=OFF \
-            -DHDF5_BUILD_EXAMPLES=OFF -DHDF5_BUILD_FORTRAN=OFF -DCMAKE_C_COMPILER=mpicc ..
+            -DHDF5_BUILD_EXAMPLES=OFF -DHDF5_BUILD_TOOLS:BOOL=OFF \
```

Comment:
What's the reason for turning tools off? I'm assuming we'll eventually turn this back on when we're at a point we can test the tools against VOLs with CMake.

Reply:
Just to speed up the builds, when we have testing we can turn it back on.

Reply:
The biggest impact will probably come from turning off static libs rather than the tools.